frontend/src/lib/components/Users/UserFilter.svelte

```diff
@@ -52,7 +59,19 @@
 >
   <svelte:fragment slot="activeFilters" let:activeFilters>
     {#each activeFilters as filter}
-      {#if filter.key === 'usersICreated' && filter.value}
+      {#if filter.key === 'userType' && filter.value}
+        <ActiveFilter {filter}>
+          {#if filter.value === 'admin'}
+            <Icon icon="i-mdi-shield-lock-outline" color="text-warning" />
+            {$t('admin_dashboard.user_filter.user_type.admin')}
+          {:else if filter.value === 'nonAdmin'}
+            <Icon icon="i-mdi-shield-lock-open-outline" />
```

Comment:
If the admin icon is a shield, then an unlocked padlock icon won't make much sense for non-admins. I suggest changing this to i-mdi-account, the standard "head and shoulders outline" icon. For guest users, I suggest the i-mdi-account-outline icon which gives the impression of a user account that isn't completely "filled in".